docs/timeseries/index.rst

```diff
@@ -270,31 +268,29 @@ time — this returns a |BinnedTimeSeries|::
     >>> ts_binned = aggregate_downsample(ts_folded, time_bin_size=0.03 * u.day)  # doctest: +REMOTE_DATA +IGNORE_WARNINGS
     >>> ts_binned  # doctest: +FLOAT_CMP +REMOTE_DATA
     <BinnedTimeSeries length=74>
-       time_bin_start   time_bin_size ...        pos_corr2          sap_flux_norm
```

Comment:
If you're seeing this in split diff, you might be wondering what the actual difference is (it certainly wasn't obvious to me).
What's changed is that some *horizontal* whitespace was removed on the "pix" column because row that got removed happened to have most demanding value in terms of horizontal space (namely, `-2.2798192730988376e-05`). This is slightly easier to see in unified diff view.